### DISCUSSION TITLE
为什么我安装不了，win11系统

### DISCUSSION THREAD

**Discussion by 71one** [ywmoyue/biliuwp-lite#907] [Q&A]
![Snipaste_2024-11-06_11-33-39](https://github.com/user-attachments/assets/5bc0a784-c13c-46d3-8069-d78535ec7cde)

安装的时候安装失败显示原因：应用安装程序无法安装包相关项

**Comment by ywmoyue** [comment]
请查看安装教程，或在discussion中搜索相同问题

https://github.com/ywmoyue/biliuwp-lite/discussions/895

https://github.com/ywmoyue/biliuwp-lite/discussions/839
